### PR TITLE
mwlwifi: update to version 10.3.8.0-20180615

### PR DESCRIPTION
**Pull request by anomeome**
add support for 88W8997 firmware, keep firmware out as no current device, get latest driver code

compile / test for mvebu / rango

Signed-off-by: Kabuli Chana <newtownBuild@gmail.com>

### CODE REVIEW

File: target/linux/mvebu/profiles/000-Default.mk [hunk 1]
```diff
@@ -8,7 +8,8 @@
 define Profile/Default
 	NAME:=Default Profile (all drivers)
 	PACKAGES:= kmod-mwlwifi wpad-mini swconfig mkf2fs e2fsprogs \
-		   mwlwifi-firmware-88w8864 mwlwifi-firmware-88w8964
+		   mwlwifi-firmware-88w8864 mwlwifi-firmware-88w8964 \
+		   mwlwifi-firmware-88w8997
```
Comment (hauke) [inline target/linux/mvebu/profiles/000-Default.mk]: If this FW is not used on any board, please do not add it to the default packages.

**Comment by EXSERENS** [comment]
now please upgrade to 20180615, thanks XD

**Comment by anomeome** [comment]
Dropped the default and updated to the new push

**Comment by blogic** [comment]
LGTM, @karlp @hnyman either of you have any stock in the mvebu world ?

**Comment by hnyman** [comment]
Yes, using also a wrt3200acm. Ok for me.
And likely also for 18.06

**Comment by blogic** [comment]
merged, Thanks !

**Comment by blogic** [comment]
@hnyman your call, i am a bit scared to merge stuff like this to 18.06 as we will rc1 tag today

**Comment by hnyman** [comment]
Let it bake on master for a few days first and then take that into rc2 or final in 18.06

**Pull request closed without merging**